lib/internal/streams/destroy.js

```diff
@@ -138,6 +135,13 @@ function errorOrDestroy(stream, err, sync) {
   const r = stream._readableState;
   const w = stream._writableState;
 
+<<<<<<< HEAD
+=======
+  if ((w && w.destroyed) || (r && r.destroyed)) {
+    return this;
+  }
+
+>>>>>>> stream: fix multiple destroy calls
```

Comment:
It seems the latest rebase introduces merge conflict

Reply:
Thanks for pointing that out!